Lib/test/test_asyncio/test_streams.py

```diff
@@ -63,15 +42,13 @@ def tearDown(self):
 
     @mock.patch('asyncio.streams.events')
     def test_ctor_global_loop(self, m_events):
-        stream = asyncio.Stream(mode=asyncio.StreamMode.READ,
-                                _asyncio_internal=True)
+        stream = asyncio.StreamReader(_asyncio_internal=True)
         self.assertIs(stream._loop, m_events.get_event_loop.return_value)
 
     def _basetest_open_connection(self, open_connection_fut):
         messages = []
         self.loop.set_exception_handler(lambda loop, ctx: messages.append(ctx))
-        with self.assertWarns(DeprecationWarning):
-            reader, writer = self.loop.run_until_complete(open_connection_fut)
+        reader, writer = self.loop.run_until_complete(open_connection_fut)
```

Comment:
Should the ``with self.assertWarns(DeprecationWarning):`` be added back to this location? I believe it's from the explicit passing the loop, but I'm not 100% certain. This also applies the the two similar sections below on lines 79 and 115.

Reply:
Confirmed the purpose of the deprecation warning by running this specific unit test and seeing the warning emitted from this line.